internal/provider/log_export_config_resource.go

```diff
@@ -320,6 +325,13 @@ func loadLogExportIntoTerraformState(
 		apiRegion = types.StringValue(spec.GetRegion())
 	}
 
+	var omittedChannels []types.String
+	if len(spec.GetOmittedChannels()) != 0 {
```

Comment:
nit: looks like the `len() != 0` check is unnecessary!

Reply:
You are correct, removed it.